styles/lola.css

```
@@ -7,74 +7,266 @@ Created by Olly Headey
-#invoice-amount td {margin: 0; padding:5px; vertical-align: top; }
-#invoice-amount td.item_r{text-align: right;}
-#invoice-amount td.item_l{text-align: left;white-space: normal; padding-bottom: 14px;}
-#invoice-amount td.total{text-align: right;font-weight: bold; padding: 3px 5px 3px 5px; }
-#invoice-amount span.currency {display: none}
+#invoice table#invoice-amount {
+    border-collapse:collapse;
+    clear:both;
+    width:100%;
+}
+#invoice-amount th {
+    background: #B8B885;
+    color: #fff;
+    font-weight: normal;
+    padding: 3px 5px 3px 5px;
+}
+#invoice-amount tr#discount_tr,
+#invoice-amount tr#net_total_tr,
+#invoice-amount tr#vat_tr {
+    color: #699;
+}
+#invoice-amount tr#total_tr {
+    background: #B8B885;
+    color: #fff;
+ }
+#invoice-amount tr.item td {
+    background: #fff;
+    border-bottom: 1px solid #B8B885;
+    padding-bottom: 10px;
+    padding-top: 10px;
+}
+#invoice-amount td {
+    margin: 0;
+    padding: 5px;
+    vertical-align: top;
+}
+#invoice-amount .item_r {
+    text-align: right;
+}
+#invoice-amount .item_l {
+    text-align: left;
+    white-space: normal;
+}
+#invoice-amount .total{
+    font-weight: bold;
+    line-height: 1.3;
+    padding: 3px 5px 3px 5px;
+    text-align: right;
+ }
+#invoice-amount span.currency {
+    display: none;
+}
 
-#invoice-amount .details_th{width:54%}
-#invoice-amount .details_notax_th{width:62%}
-#invoice-amount .quantity_th{width:13%}
-#invoice-amount .subtotal_th{width:15%; text-align:right}
-#invoice-amount .unitprice_th{width:10%; text-align:right}
-#invoice-amount .salestax_th{width:8%; text-align:center}
+#invoice-amount .details_th,
+#invoice-amount td.description_td {
+    width:47%;
+}
+#invoice-amount .details_notax_th {
+    width:55%;
+}
+#invoice-amount .quantity_th {
+    width:13%;
+}
+#invoice-amount .subtotal_th {
+    text-align: right;
+    width: 18%;
+}
+#invoice-amount .unitprice_th {
+    text-align: right;
+    width: 14%;
+}
+#invoice-amount .salestax_th {
+    text-align:right;
+    width: 8%;
+}
 
 /* Invoice UK VAT Summary Table */
-#invoice #invoice-uk-vat-summary{width:100%;border-collapse:collapse;margin:25px 0 0 0;clear:both;border:0; background:#fff; }
-#invoice tr#invoice-uk-vat-header th{padding:7px 0;border-top: 1px solid #ccc;}
-#invoice-uk-vat-summary td, #invoice #invoice-uk-vat-summary th{text-align:left; font-size:9pt;padding:2px 0;}
-#vat-total td, #vat-total th{border-top:1px solid #ccc;}
+#invoice #invoice-uk-vat-summary{
+    background: #fff;
+    border: 0;
+    border-collapse: collapse;
+    clear: both;
+    margin: 25px 0 0 0;
+    width: 100%;  
+}
+#invoice tr#invoice-uk-vat-header th{
+    border-top: 1px solid #ccc;
+    padding: 7px 0;
+}
+#invoice-uk-vat-summary td,
+#invoice #invoice-uk-vat-summary th {
+    font-size:9pt;
+    padding:2px 0;
+    text-align:left;
+}
+#vat-total td, #vat-total th {

```

Separating these to their own lines
```suggestion
#vat-total td,
#vat-total th {
```